helm_resource/Tiltfile

```diff
@@ -95,20 +95,16 @@ def helm_resource(
   if flags:
     apply_cmd.extend(flags)
 
+  dupe_deps = {}
   for i in range(len(image_deps)):
+    env['TILT_IMAGE_DEP_%s' % i] = image_deps[i]
```

Comment:
Is this var used or supposed to be used? I also notice that the help apply helper has a reference to `TILT_IMAGE_%s`, are they supposed to be the same?

Reply:
ah, good call, i think we can remove this